ds-cr/description.md

```diff
@@ -3,7 +3,8 @@ to apply good practices that enable open and reproducible research.
 The workshop focuses on building modular, reusable, maintainable, sustainable, reproducible,
 testable, and robust software. This will allow you to more easily organize, maintain and share your data. 
 The participants should be familiar with programming and regularly write code for their research, 
-but no extensive expertise or knowledge of specific tools are required.
+but no extensive expertise or knowledge of specific tools are required. 
+The main themes that are addressed are generically applicable, but please note that a lot of the exercises and demonstrations are in Python.
```

Comment:
That seems clear to me. Can you also add that line to the description without optional git introduction? https://github.com/esciencecenter-digital-skills/workshop-metadata/blob/fixes-%2394/ds-cr/description-no-option.md

Reply:
Good point, I just did and opened a new pull request #108. I will close this one.